index.js

```diff
@@ -52,6 +52,13 @@ module.exports = input => {
 		};
 	}
 
+	if (check([0x3C, 0x3F, 0x78, 0x6D, 0x6C, 0x20])) {
```

Comment:
What if there's an empty line before `<?xml version="1.0" encoding="UTF-8"?>`?

Reply:
It does not work, but the standard specify that an xml document has to start with the xml declaration ( https://www.w3.org/TR/xml/#sec-prolog-dtd ) and new lines are converted in a special character (https://www.w3.org/TR/xml/#sec-line-ends), so it is like if the document is not beginning with the xml declaration.

Reply:
Can you use the `toBytes()` method here for readability reasons?

https://github.com/sindresorhus/file-type/blob/4408621b5b8ea17911306be696c6fa1201bce34a/index.js#L2

Reply:
I tried to use it, but it does not return the correct magic number, the returned one is correct except for the last element, which is 0X2F instead of 0X20, if I slice the returned magic buffer ignoring the last element it works, but i'm not sure that this will always work fine.

Reply:
I don't understand why that shouldn't work
```
Array.from('<?xml ').map(c => c.charCodeAt(0))
(6) [60, 63, 120, 109, 108, 32]

[0x3C, 0x3F, 0x78, 0x6D, 0x6C, 0x20]
(6) [60, 63, 120, 109, 108, 32]
```

Reply:
well sorry I was using the wrong offset